frontend/src/app/pages/MainPage/pages/SourcePage/SourceDetailPage/index.tsx

```diff
@@ -481,21 +481,20 @@ export function SourceDetailPage() {
               </Form.Item>
               {dataProviderConfigTemplateLoading && <LoadingOutlined />}
 
-              {(providerType !== 'FILE' || formType === CommonFormTypes.Edit) &&
```

Comment:
@lyp000119  what is provider type file and format type edit, is it necessary？

Reply:
Fyi, I removed it as I see the condition is wrong. On demo site, if I start create a new FILE source, the form does not appear due to this condition. Not sure the reason for having it in the existing code.

![image](https://user-images.githubusercontent.com/100477931/179700484-33c0fed3-274b-4142-ae78-35e84e0fe81a.png)

Reply:
> @lyp000119 什么是provider type file和format type edit，有必要吗？

It can be modified like this, no problem

Reply:
👍🏿 Good